internal/pkg/model/receiver.go

```diff
@@ -33,3 +34,169 @@ type Receiver struct {
 	Name      string `json:"name" validate:"required,min=1,max=40"`
 	Secret    string `json:"secret" validate:"required,len=48"`
 }
+
+// To add a new column type `Foo`:
+// - create a struct for it: `type ColumnFoo struct{}`
+// - create a type name constant for it: `columnIDFoo = "foo"`
+// - add it to `columnToType` and `typeToColumn`
+// - implement `MappedColumn` for `ColumnFoo`
+
+type MappedColumns []MappedColumn
```

Comment:
Myslim, ze by to malo byt v samostatnom pkg `column`, kedze vsetko je tu prefixovane s `Column`, bude to aj samostatny adresar, tj. tie definicie jednoduchsie najdes. Minimalne by to mal byt aspon samostatny suboru `column.go`, idealne ten samostatny pkg `column/{id,datetime...}.go`.

Reply:
Uprednostnil by som `Columns/Column` pred `Mapped....`, ine `columns` tu nemame.

Reply:
No jo, ale v CLI bude dřív či později podpora Storage tabulek. A i když by tu v `model` packagi neměly přímo definice, tak se to imho bude plést a bude se na první pohled nabízet že tyhle struktury patří k tomu.

Reply:
Aha, dobra poznamka, to mi nedoslo v akom je to pkg.
Imho sa `model` chceme zbavit.

Mohol by to byt package `service/buffer/model`.

Toto je model pre `keboola-as-code` (CLI/templates/proxy api/meta store), casom tomu najdeme lepsie miesto.
Nie je dovod tieto `models` spajat.

Reply:
👍

Reply:
(doteraz bol problem kde ten model package dat, ale teraz je to uz podelene po tych services)

Reply:
Ok, přesunu to

Reply:
Přesunuto a přejmenováno na `Column`